packages/optimizer/lib/fetchRuntimeParameters.js

```diff
@@ -88,30 +88,30 @@ async function initValidatorRules(runtimeParameters, customRuntimeParameters, co
     config.log.verbose(error);
   }
   try {
-    runtimeParameters.bentoComponentInfo =
-      customRuntimeParameters.bentoComponentInfo ||
-      config.bentoComponentInfo ||
-      (await fetchBentoComponentInfoFromCache_(config));
+    runtimeParameters.componentVersions =
+      customRuntimeParameters.componentVersions ||
+      config.componentVersions ||
+      (await fetchComponentVersionsFromCache_(config));
   } catch (error) {
-    config.log.error('Could not fetch bento component info');
+    config.log.error('Could not fetch latest component versions from amp.dev');
```

Comment:
Nit, if it can continue, it feels like it should be a warning rather than an error

Reply:
done